frontend/src/containers/sidebar-form/SelectMenuContainer.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import useFetch from '../../lib/useFetch';
+import SelectMenuList from './components/SelectMenuList';
+
+const SelectMenu = styled.div`
+  background-color: white;
+  border: 1px solid lightgray;
+  border-radius: 3px;
+  color: black;
+  overflow: hidden; 
+  z-index: 10;
+`;
```

Comment:
```suggestion
const SelectMenu = styled.div`
  background-color: white;
  border: 1px solid lightgray;
  border-radius: 3px;
  color: black;
  overflow: hidden; 
  z-index: 10;
  display:  ${(props) => (props.open? 'block' : 'none')};
  position: 'absolute';
  top: '140%';
  right: '0',
```
props 활용하시면 style 인라인으로 적용하지 않아도 될 것 같습니다 !